content/docs/studio/user-guide/model-registry/use-models.md

```diff
@@ -1,17 +1,29 @@
 # Use models
 
 Whether you need to download your models to use them, or you're looking to set
-up some automation in CI/CD to deploy them, Iterative Studio provides these
+up some automation in CI/CD to deploy them, DVC Studio provides these
 capabilities.
 
 ## Download models
 
 If your model file is DVC-tracked, you can download any of its registered
-versions using Studio [REST API], `dvc get`, or DVC [Python API].
+versions using the DVC Studio [REST API], DVC [Python API], use the
+`dvc artifacts get` command or directly download your model from DVC Studio.
 
-### Using Studio API
+Prerequisites:
 
-Using Studio API will require you to add your remote storage secrets to
+- The DVC Studio project you like to download your model from needs access to
+  your [remote storage credentials].
+- Access to your [DVC Studio Access token].
+
+[remote storage credentials]:
+  /doc/studio/user-guide/experiments/configure-a-project#data-remotes--cloud-storage-credentials
+[DVC Studio Access token]:
+  /doc/studio/user-guide/account-management#studio-access-token
+
+### Using DVC Studio API
```

Comment:
I think this repeats too much of what is said in other docs. I would prefer we basically copy and paste the tabs we have in the UI (we can even use tabs again here). WDYT?

Reply:
PTAL at https://github.com/iterative/dvc.org/pull/4939/commits/907d805aa486c4515fcd49aa29cb3f5d75a292b9